scripts/preview_docc.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the Swift Metrics API open source project
+##
+## Copyright (c) 2018-2019 Apple Inc. and the Swift Metrics API project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of Swift Metrics API project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+xcrun swift package --disable-sandbox preview-documentation --target "$1"
```

Comment:
xcrun seems wrong, only available on macOS

Reply:
```suggestion
swift package --disable-sandbox preview-documentation --target "$1"
```

yep, can remove `xcrun`.

Reply:
Gotcha thanks